addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -3,15 +3,54 @@
 import { Component } from "@odoo/owl";
 import { useRtc } from "@mail/rtc/rtc_hook";
 import { isMobileOS } from "@web/core/browser/feature_detection";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { _t } from "@web/core/l10n/translation";
 
 export class CallActionList extends Component {
+    static components = { Dropdown, DropdownItem };
     static props = ["thread", "fullscreen", "compact?"];
     static template = "mail.CallActionList";
 
     setup() {
         this.rtc = useRtc();
     }
 
+    get actions() {
+        const acts = []
```

Comment:
Make sure you use prettier/eslint according to `web/tooling` config in you IDE.
Here, it should tell that `;` is missing at EOL.
This helps formatting code in a way that is consistent in most Odoo JS files, and reviews do not waste time for formatting problems :)